tvac/client.py

```diff
@@ -0,0 +1,66 @@
+import requests
+
+VARIANT_ENDPOINT = "http://rest.ensembl.org/vep/human/hgvs/{variant}" 
+
+class EnsemblError(Exception):
+    def __init__(self, status_code, error_text):            
+        # Call the base class constructor with the parameters it needs
+        super().__init__(error_text)
+            
+        self.status_code = status_code
+        self.error_text = error_text
+
+    def __str__(self):
+        return f"Annotation website returned bad response\nstatus_code:{self.status_code}\nerror:{self.error_text}"
+
+class VariantAnnotationClient:
+    def __init__(self, variant_endpoint=VARIANT_ENDPOINT):
+        self.variant_endpoint = variant_endpoint
+
+    def parse_variants(self, filename):
```

Comment:
I thought about adding an error handler to the file reading but decided that there's no value to adding custom handling in addition to the default exception throwing.